server/chat-commands/admin.ts

```diff
@@ -929,8 +929,14 @@ export const commands: ChatCommands = {
 	async eval(target, room, user, connection) {
 		if (!this.canUseConsole()) return false;
 		if (!this.runBroadcast(true)) return;
+		const logRoom = Rooms.get('upperstaff') || Rooms.get('staff');
 
-		if (!this.broadcasting) this.sendReply(`||>> ${target}`);
+		if (this.message.startsWith('>>') && room) {
+			this.broadcasting = true;
+			this.broadcastToRoom = true;
+		}
+		this.sendReply(`|html|<table border="0" cellspacing="0" cellpadding="0"><tr><td valign="top">&gt;&gt;&nbsp;</td><td>${Chat.getReadmoreCodeBlock(target)}</td></tr><table>`);
```

Comment:
Sadly, CSS doesn't really give us an easy way to do cell styling without a `<style>` tag or anything like that. This is unfortunately still the best way to do an invisible table.